CHANGES.txt

```diff
@@ -39,6 +39,10 @@ Breaking Changes
 Changes
 =======
 
+ - Added new "password" authentication method which is available for connections
```

Comment:
I think it would make sense to merge this directly with https://github.com/crate/crate/pull/6617/files#diff-bb3c4deeba77c2bbfd893f2e90fecc77R68

Reply:
But maybe mention the deprecation of `X-USER` ?